app/src/main/res/xml/developer_preferences.xml

```diff
@@ -385,8 +385,8 @@
 
         <org.wikipedia.settings.IntPreference
             style="@style/IntPreference"
-            android:key="ab_test_exploreVsHome"
-            android:title="Whether the user sees a Home icon/label, or Explore icon/label." />
+            android:key="ab_test_suggestedEditsSnackbar"
+            android:title="Whether the user sees a Snackbar action to 'Get Started' with suggested edits, after an unrelated edit." />
```

Comment:
Any reason why we keep this? It was removed in #1865

Reply:
whoops, not sure why this made its way back.